README.md

```diff
@@ -51,11 +58,18 @@ To run the service locally for debugging override the application.yml configurat
 notify:
   apiKey: dummykey-ffffffff-ffff-ffff-ffff-ffffffffffff-ffffffff-ffff-ffff-ffff-ffffffffffff
   baseUrl: http://localhost:8917
-  templateEnglish: 21447bc2-e7c7-41ba-8c5e-7a5893068525
-  templateWelsh: ef045f43-ffa8-4047-b8e2-65bfbce0f026
-  templateWelshAndEnglish: 23f96daf-9674-4087-acfc-ffe98a52cf16
-  templateNorthernIreland: 1ccd02a4-9b90-4234-ab7a-9215cb498f14
-  senderId: b7e75a6c-8f7f-4264-8b34-38bb5831270e
+  templateEnglishHouseHold: ce1e545e-f50f-455b-a394-88b49a36fa0c
+  templateWelshHouseHold: 67dc02ff-a667-4b21-8e4e-61dd28936b8c
+  templateWelshAndEnglishHouseHold: 2375c493-359d-4712-938e-bf97a641f18f
+  templateNorthernIrelandHouseHold: 5d985237-b446-492a-bd0a-f7368265282c
+  templateEnglishIndividualResponse: b7ec0cde-b55b-460a-a78a-cb80767acdca
+  templateWelshIndividualResponse: b2db05f0-d7d1-4b88-b3de-658495bd8a47
+  templateWelshAndEnglishIndividualResponse: f6de09f2-4d4c-4f62-82e3-b1a24e0cf910
+  templateNorthernIrelandIndividualResponse: d490af7d-300a-4eb8-a961-cb4de54332ee
+  templateEnglishCe: 21f22f8d-2642-444e-9d13-a54b87647a93
+  templateWelshCe: 2c12a125-4035-4b81-9988-204e02e759a5
+  templateWelshAndEnglishCe: b2b9e650-cb22-49d7-b5da-95169e13ea12
```

Comment:
I think it would be easier to use the fulfilment codes here rather than a description of each one

Reply:
Still needs addressing, it's been changed in the README but not in the code, I should have been clear that I meant everywhere we were using these template names we should drop them and just use the treatment codes.